### PR TITLE
Node webshot

### PR DESCRIPTION
**Pull request by mdiehr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
   "homepage": "https://github.com/mdiehr/mtgnewsbot#readme",
   "dependencies": {
-    "html2png": "1.1.2",
     "jimp": "0.2.28",
     "lodash.merge": "4.6.0",
     "svg2png": "4.1.1",
     "tracery-grammar": "2.7.3",
-    "twit": "2.2.9"
+    "twit": "2.2.9",
+    "webshot": "^0.18.0"
```
Comment (jwoolley) [inline package.json]: probably fix this to this specific build

File: src/index.js [hunk 2]
```diff
@@ -33,22 +32,11 @@ headlines.forEach(headline => {
...
-            })
-          .catch(err => { 
-            console.log('\n *** Failed to create trimmed png:');
-            console.log(err);
-          });
-			}
-		});	  
+		RenderImage.fromHtml(html, outputPath)
+			.then(localFilePath => console.info('\n *** Image saved to ' + localFilePath))
+			.catch(e => console.error('Failed to download image: ' + e));
```
Comment (jwoolley) [inline src/index.js]: error string should rather be something like "failed to render image"

File: package.json (hunk 1 above)
Comment (mdiehr) [inline package.json]: oh yes, of course

File: src/index.js (hunk 2 above)
Comment (mdiehr) [inline src/index.js]: done

**Pull request merged**